source/bigint.c

```diff
@@ -483,3 +483,235 @@ void aws_bigint_negate(struct aws_bigint *bigint) {
         bigint->sign *= -1;
     }
 }
+
+static void s_aws_bigint_trim_leading_zeros(struct aws_bigint *bigint) {
```

Comment:
This appears to be trimming trailing zeroes?

Reply:
Well, logically they are leading, physically they are trailing.  I prefer the logical viewpoint to be honest.

Reply:
oh I see now